Apps/phgsgmail/gsgmail_connector.py

```diff
@@ -336,6 +434,75 @@ def _handle_run_query(self, param):
 
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    def _handle_get_email(self, param):
+
+        # Implement the handler here, some basic code is already in
```

Comment:
nit: The comment sounds redundant, right?

Reply:
probably a copy-paste thing. See above comment, this is just a PR from the PS team that I refactored and added automation. Notice in the action handler function above the same comment exists